vms/avm/vm.go

```diff
@@ -590,40 +572,6 @@ func (vm *VM) lookupAssetID(asset string) (ids.ID, error) {
 // before its state changes are applied.
 // Invariant: any error returned by onAccept should be considered fatal.
```

Comment:
Not: comment is out of date

Reply:
updated https://github.com/ava-labs/avalanchego/pull/3872/commits/4b8010d245fb3a948f402f21428086526e6ac3f5